docs/update.md

```diff
@@ -1,13 +1,20 @@
 # `packit propose-update`
 
-This is a detailed documentation for the update functionality of packit.
+This is a detailed documentation for the update functionality of packit. The
+command creates a new pull request in Fedora using a selected or latest
+upstream release.
 
 
 ## Requirements
 
+* Upstream git repository on Github.
```

Comment:
GitHub is MUST? I know, ogr has support only for GitHub and GitLab. But it can be used also for others, can't it?

Reply:
Ogr now supports only Github and Pagure.

For packit, we are using Github as Upstream and Pagure for Downstream. (We are choosing directly the specific implementation.) It would be easy to make it more general in the future, but not now.

Reply:
to be honest, I'm not 100% sure if the upstream repo needs to be github for propose-update; but since our scope was indeed limited to github from the beginning, let's market it as such; otherwise we could get a ton of RFEs for different forges which we would not have time to implement

Reply:
> to be honest, I'm not 100% sure if the upstream repo needs to be GitHub for propose-update; 

You're right, I thought we are loading config with `GithubService`/`GithubProject`, but that's not the case now. (It's used only for `watch` methods.)